plugins/Jobs.cpp

```diff
@@ -920,7 +922,7 @@ void BedrockJobsCommand::process(SQLite& db) {
 
         // Verify there is a job like this and it's running
         SQResult result;
-        if (!db.read("SELECT state, nextRun, lastRun, repeat, parentJobID, json_extract(data, '$.mockRequest'), retryAfter "
+        if (!db.read("SELECT state, nextRun, lastRun, repeat, parentJobID, json_extract(data, '$.mockRequest'), retryAfter, json_extract(data, '$.originalNextRun') "
```

Comment:
Do we need to clear the originalNextRun at some point or it is not necessary?
I _think_ we don't because:
- If the job did not have a repeat, it will be deleted when finished
- If it has a repeat, it will get re-scheduled and the originalNextRun re-set when retrieved in getjob

Reply:
Agreed, thought the same